recipe/meta.yaml

```diff
@@ -37,10 +40,11 @@ test:
   requires:
     - pip
   commands:
-    - python -m pip check
+    - pip check
     - jsonschema --version
     - jsonschema --help
 
+
 about:
   home: https://github.com/Julian/jsonschema
```

Comment:
This could also be set to https://github.com/python-jsonschema/jsonschema

Reply:
Done, thanks!